makefile

```diff
@@ -181,8 +185,12 @@ examples: $(EXS)
 # compile examples; note absolute .so path so executable anywhere, dep libs not needed to be listed...
 $(EX): $(EX).o $(DYNLIB)
 	$(CXX) $(CXXFLAGS) $(EX).o $(ABSDYNLIB) -o $(EX)
+$(EX_32): $(EX_32).o $(DYNLIB)
```

Comment:
Can't these be handled by pattern rules (in case we want to add more in the future)?

Reply:
Sure, I was trying not to stomp on Alex's makefile too much, but I guess since I did the FORTRAN targets I should hit this one too. pushed.

Note both the `fortran` and `example` only execute once using the scheme in this makefile.  You guys are welcome to change that to your taste (using phony or force etc).  Probably best to wait until all the bindings are where you want then optimize the makefile to be consistent.  Hope these changes aren't too much.  Garrett